lib/entities/player.rb

```diff
@@ -0,0 +1,22 @@
+class Player
+  include Validation
+
+  attr_reader :name
+  attr_accessor :errors_store
+
+  MAX_LENGTH = 20
+  MIN_LENGTH = 3
+
+  def assign_name(name)
+    @errors_store = []
+    validate_name(name) ? @name = name : @errors_store << I18n.t(:when_wrong_name)
```

Comment:
if you have assignment in just one condition - don't use ternary operator. Use guard clause